cmd/remove.go

```diff
@@ -33,6 +35,10 @@ func RemoveCommand() cli.Command {
 			Name:  "local",
 			Usage: "Deploy Kubernetes cluster locally",
 		},
+		cli.BoolFlag{
+			Name:  "dind",
+			Usage: "Remove Kubernetes cluster deployed in dind mode",
```

Comment:
flag name "dind" doesn't really correlate to it's usage - it is used to remove the cluster?

Reply:
can't we figure out in which mode the cluster was deployed, w/o passing the flag? ideally the flag should be passed on create only

Reply:
there is no way to identify the mode of rke installation of the nodes, each mode has different dialer and healthcheck dialer, that's why we use:
```
rke up --dind
rke remove --dind
```
and for local
```
rke up --local
rke remove --local
```